### ISSUE TITLE
Settings - Header dropdown

### ISSUE THREAD

**Issue by crsfalzon** [SimplyStaking/panic#223]
### User Story

As a node operator, I want to easily access a settings section, so I can review/update my PANIC configuration.

### Description

The scope of this ticket is limited to the Settings dropdown in the header.

### Requirements
The settings dropdown menu has two entries:

- Chains
- Channels

![Update chain 1](https://user-images.githubusercontent.com/89395527/171124901-37d68a9b-be0c-4551-a03b-42177568a408.PNG)

### Blocked by
Not dependent on another ticket to be tackled.

### Acceptance criteria
 **Scenario**: The node operator is on a dashboard page and wants to review/update his configuration (change a specific alert severity type, node, add a sub-chain,...)
**Given**: The node operator is on a dashboard page
**When**: He clicks on the Settings button in the header
**Then**: He is presented with two options - Chains and Channels
